col.h

```diff
@@ -0,0 +1,11 @@
+struct Square {
+	//Center position
+	float x, y;
```

Comment:
Should that be center position? Wouldn't it be way easier to work with the corner with lowest `x` and `y`?